src/components/policy/policy_external/test/policy_manager_impl_ptu_test.cc

```diff
@@ -52,8 +52,9 @@ using ::testing::_;
 using ::testing::Return;
 using ::testing::ReturnRef;
 
-TEST_F(PolicyManagerImplTest,
-       TiggerPTUForNaviAppInCaseNoCertificateExistsInPolicyTable) {
+TEST_F(
+    PolicyManagerImplTest,
+    /*DISABLED_*/ TiggerPTUForNaviAppInCaseNoCertificateExistsInPolicyTable) {
```

Comment:
@pold500 Redundant comment. Also please fix tests and add new one according your changes